src/classes/class-shortcodes.php

```diff
@@ -124,10 +125,36 @@ function store_url( $props, $content = null ) {
 		 * @return null
 		 */
 		function policy_page( $props, $content = null ) {
-			$url     = get_permalink( get_option( 'wp_page_for_privacy_policy' ) );
-			$content = sprintf( '<a href="%s">%s</a>', $url, $url );
-			return $content;
+
+			return $this->woo_permalink( [ 'option' => 'wp_page_for_privacy_policy' ] );
 		}
 
+		/**
+		 * Create a permalink based on post/page/product id.
+		 *
+		 * @param $props
+		 * @param null  $content
+		 * @return null
+		 */
+		function woo_permalink( $props, $content = null ) {
```

Comment:
Do we really need `$content` here?

Reply:
I suspect someone will want to have links translatable or to have some custom styling etc. 

`[woo-permalink option="page_id"]<a href="%s">Moja stran</a>[/woo-permalink]`

Reply:
I see. Makes sense.

The only edge case I see where it can throw an error is if **$content** has `%s` more than twice.